site/style.css

```
@@ -1,52 +1,75 @@
-  --planned: #807060;
-  --warn: #F1C40F;
-  --shadow-color: #ff6b6b;
-  --shadow-hard: 4px 4px 0 var(--shadow-color);
-  --shadow-hard-lg: 6px 6px 0 var(--shadow-color);
-  --dot-color: rgba(255, 255, 255, 0.04);
-  --overlay-bg: rgba(13, 13, 24, 0.88);
-  --modal-bg: #141428;
-  --header-bg: rgba(13, 13, 24, 0.94);
-  --code-bg: #111122;
+  --font-display: 'VT323', ui-monospace, 'JetBrains Mono', monospace;
+  --font-body: 'Source Serif 4', 'Source Serif Pro', 'Iowan Old Style', Georgia, serif;
```

Stylelint (17.10.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)